ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/disconnected/private/provision/cucushift-installer-rehearse-gcp-ipi-disconnected-private-provision-chain.yaml

```diff
@@ -11,6 +11,7 @@ chain:
   - ref: ipi-conf-telemetry
   - ref: ipi-conf-gcp
   - ref: ipi-conf-mirror
+  - ref: ipi-conf-manual-creds-remove-unnecessary-creds
```

Comment:
I think this workflow does not need this step.

Reply:
Updated.